conda_smithy/configure_feedstock.py

```diff
@@ -1581,13 +1576,13 @@ def clear_variants(forge_dir):
 
 
 def get_common_scripts(forge_dir):
-    for old_file in ["run_docker_build.sh", "build_steps.sh"]:
+    for old_file in ["run_docker_build.sh", "build_steps.sh", "run_osx_build.sh"]:
```

Comment:
makes sure that `_render_template_exe_files` checks the osx build script for different renderings on different platforms, not that that should ever happen.